SteamcLog/Classes/Helper/LogLevelPreset.swift

```diff
@@ -9,57 +9,55 @@
 import Foundation
 
 public enum LogLevelPreset: String, Codable {
-    /// Disk: verbose, system: verbose, remote: none, analytics: disabled
-    case firehose
-    /// Disk: none, system: debug, remote: none, analytics: disabled
-    case develop
-    /// Disk: verbose, system: none, remote: warn, analytics: enabled
-    case releaseAdvanced
-    /// Disk: none, system: none, remote: warn, analytics: enabled
+
+    case debugVerbose
+    case debug
     case release
+    case releaseAdvanced
 
     var global: LogLevel {
         switch self {
-        case .firehose: return .info
-        case .develop: return .info
-        case .releaseAdvanced: return .info
+        case .debugVerbose: return .info
+        case .debug: return .info
         case .release: return .warn
+        case .releaseAdvanced: return .info
         }
     }
 
-    var sentry: LogLevel {
+    var remote: LogLevel {
         switch self {
-        case .firehose: return .none
-        case .develop: return .none
-        case .releaseAdvanced: return .info
-        case .release: return .warn
+        case .debugVerbose: return .none
+        case .debug: return .none
+        case .release: return .info
+        case .releaseAdvanced: return .warn
         }
     }
 
-    var file: LogLevel {
+    var disk: LogLevel {
         switch self {
-        case .firehose: return .verbose
-        case .develop: return .none
-        case .releaseAdvanced: return .verbose
-        case .release: return .none
+        case .debugVerbose: return .verbose
+        case .debug: return .debug
+        case .release: return .info
+        case .releaseAdvanced: return .debug
         }
     }
 
-    var system: LogLevel {
+    var console: LogLevel {
         switch self {
-        case .firehose: return .verbose
-        case .develop: return .debug
-        case .releaseAdvanced: return .none
+        case .debugVerbose: return .verbose
+        case .debug: return .debug
         case .release: return .none
+        case .releaseAdvanced: return .none
         }
     }
 
+    // TODO 2021-09-16: If analytics are by default no longer supported, should we remove this?
```

Comment:
Yeah, I think we should just chop this out

Reply:
I have a ticket to remove track (https://github.com/steamclock/steamclog-swift/issues/96), so I'll pull this code out then as well.